tests/FSharp.Core.UnitTests/SurfaceArea.fs

```diff
@@ -9,2468 +9,2491 @@ type SurfaceAreaTest() =
     [<Fact>]
     member this.VerifyArea() =
         let expected = @"
-Microsoft.FSharp.Collections.Array2DModule: Int32 Base1[T](T[,])
```

Comment:
Do you know why this diff turned out so large?  It makes it hard to review?

Reply:
I have no idea. I have replaced the surface output from my PC and how the tests runs fine on both my mac and my windows machine but the tests in the repo still fail. I have .NET 6 previews installed but I don't think the build system uses them?

I also thought about reverting this commit and just manually adding the added methods but i was not sure if the order was important. 

I'd appreciate any help or suggestions on how to resolve this test.